src/index.js

```diff
@@ -0,0 +1,13 @@
+import './index.css';
+import addHtml from './modules/addHtml.js';
+import btnAdd from './modules/buttonAdd.js';
+import refreshBtn from './modules/refresh.js';
+import { clearFunction } from './modules/check.js';
```

Comment:
There is a good use of ES6 javascript modules which is a good practice.

Reply:
Thank you @Kelvin-Ben for this review. I hadn't noticed. :+1: